sdks/java/core/src/main/java/org/apache/beam/sdk/io/TextIO.java

```diff
@@ -91,6 +96,20 @@
  * }</pre>
  */
 public class TextIO {
+  private static class IdentityFormatter<T> implements SerializableFunction<T, T> {
+    @Override
+    public T apply(T input) {
+      return input;
+    }
+  }
+
+  private static class ExtractValueFormatter<K, V> implements SerializableFunction<KV<K, V>, V> {
```

Comment:
Done for identity. Will hold off on this one until we decide whether we'll be keeping the KV helper.